src/components/query-panel/view/delete-panel.html

```diff
@@ -0,0 +1,27 @@
+<uib-accordion close-others="oneAtATime">
+  <div uib-accordion-group class="panel-default">
+    <uib-accordion-heading>{{ctrl.name}} {{ctrl.description}}</uib-accordion-heading>
+    <div class="row">
+      <div class="col-sm-3 col-lg-2">
+        <div class="form-group">
+          <span>Enter key: </span>
+          <input class="form-control" type="text" id="{{ctrl.name}}-input-text"/>
+        </div>
+      </div>
+      <div class="col-sm-9 col-lg-6">
+        <button type="button" class="btn btn-default execute-button" ng-click="ctrl.executeAction()">Execute</button>
```

Comment:
I may be missing something but it seems like the `*-panel.html` files are essentially the same. Can we not just have one generic html template for all operations?

Reply:
+1 Ideally. I did not want to overoptimize right away. Something to strive to. Still waiting for UI feedback from Catherine. We can decide then what to do. But for the time being, I left the option to customize each screen individually.